File: Parte1/desafios de logica/desafios1/Desafio.kt
object ReceitaFederal {
    fun calcularImposto(salario: Double): Double {
        val aliquota = when {
            (salario >= 0 && salario <= 1100) -> 0.05
            (salario > 1100 && salario <= 2500) -> 0.10
            else -> 0.15
        }
        return aliquota * salario
    }
}

fun main() {
    val valorSalario = readLine()!!.toDouble;
    val valorBeneficio = readLine()!!.toDouble;

    val valorImposto = ReceitaFederal.calcularImposto(valorSalario);
    val saida = valorSalario - valorImposto + valorBeneficio;

    println(String.format("%.2f", saida));
}

File: Parte1/desafios de logica/desafios1/Desafio.py

def calcular_imposto(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 0.05
    elif (salario > 1100 and salario <= 2500):
        aliquota = 0.10
    else:
        aliquota = 0.15

    return aliquota * salario

valorSalario = float(input("Qual o valor do seu salario?"))
valorBeneficio = float(input("Qual o valor do seu beneficio?"))
valorImposto = calcular_imposto(valorSalario)
saida = valorSalario - valorImposto + valorBeneficio
print(f'{saida:.2f}')